RPIPs/RPIP-X.md

```diff
@@ -0,0 +1,308 @@
+
+To give an example, the oDAO has the power to make changes to the Rocket Pool Protocol due to the permissions trustedNodes (in aggregate) hold in the protocol contracts, and because the limitations to the oDAO ratified by the pDAO are self-applied by the oDAO.
+
+Best expressed in the phrase: ‘Are we able to do X?’
+
+#### Authority
+
+Authority in this document is used to refer to the right to do something. This refers to the implicit and explicit decisions made by the community at large as to ‘who do we want to do X’.
+
+Authorities are a bit complex in that they often stem from a string of decision-making powers, some explicit and some explicit. This document seeks to document what clear authorities exist based on approved governance actions and their ramifications, but may not be able to capture the full intricacy of who has the "right" to take an action.
+
+A principal or agent has the authority to do X if the Rocket Pool community has explicitly or implicitly agreed that that principal or agent is allowed or supposed to be doing X.
+
+To give an example, the pDAO has the authority to make changes to the Rocket Pool Protocol due to the shared understanding of the community that registeredNodes are the ‘principals’ of the Rocket Pool protocol.
+
+Best expressed in the phrase: ‘Are we allowed to do X?’
+
+### Principals and Agents
+
+#### oDAO governance
+
+##### Description
+
+OracleDAO governance is defined as successful on-chain governance proposals voted on by oDAO members running oDAO nodes.
+
+##### Powers
+
+* The oDAO has the power to replace any contract (except RocketStorage) in the Rocket Pool protocol via the oDAO upgrade power.
+* The oDAO has the power to modify any non-protected parameter in the Rocket Pool protocol via the oDAO upgrade power.
+* The oDAO has the power to call core contract functions protected by the generic ‘onlyLatestNetworkContract’ required via the oDAO upgrade power. This has several implications, most notably:
+  * The ability to withdraw RPL and ether from the RocketVault contract.
+  * The ability to withdraw ether from the RocketSmoothingPool contract.
+  * The ability to set the Minipool penalty within the bounds of the maximum penalty.
+* The oDAO has the power to modify oDAO settings via the oDAO settings pathway.
+* The oDAO has the power to take actions that modify the membership of the oDAO.
+* The oDAO has the power to determine the split of RPL inflation via consensus on a rewards Merkle tree. [[GitHub](https://github.com/rocket-pool/rocketpool-research/blob/v6/Merkle%20Rewards%20System/rewards-calculation-spec.md)]
+* The oDAO has the power to withdraw ETH at no cost from the deposit pool contract, via manipulation of the rETH exchange rate. [[Etherscan](https://etherscan.io/address/0xDD3f50F8A6CafbE9b31a427582963f465E745AF8)]
```

Comment:
I'd probably talk about the power to set the exchange rate, and have these as sub-bullets.

- The oDAO has the power to set the protocols rETH exchange rate.
  - Note that manipulating this value would allow for withdrawing ETH at no cost (from both the deposit pool contract and rETH token contract).